python/tk_multi_publish2/dialog.py

```diff
@@ -297,8 +300,18 @@ def _update_item_thumbnail(self, pixmap):
         thumbnail pixmap
         """
         if not self._current_item:
-            raise TankError("No current item set!")
-        self._current_item.thumbnail = pixmap
+            # this is the summary item
+            self._summary_thumbnail = pixmap
+            if pixmap != None:
+               # update all items with the summary thumbnail
+               for top_level_item in self._plugin_manager.top_level_items:
+                   top_level_item.thumbnail = self._summary_thumbnail
+                   top_level_item._set_thumbnail_enabled(True)
```

Comment:
We shouldn't be calling protected methods on an object. Also, thumbnail setting is explicitly disabled on the items in some plugins. I don't think we should force them to be enabled here. Perhaps only do the previous line if the thumbnail is enabled?